Procfile

```diff
@@ -0,0 +1 @@
+web: java -Dserver.port=$PORT $JAVA_OPTS -javaagent:newrelic/newrelic.jar -jar target/play-ping-pong.jar
```

Comment:
Procfile is specific to Heroku: https://devcenter.heroku.com/articles/procfile
It is used to customize how the application runs